wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxDataSource.swift

```diff
@@ -91,22 +91,20 @@ class MessageToolboxDataSource {
 
     /**
      * Updates the contents of the message toolbox.
-     * - parameter forceShowTimestamp: Whether the timestamp should be shown
      * - parameter widthConstraint: The width available to rend the toolbox contents.
+     * - Returns: A boolean to either update the content of the message toolbox or not
      */
-
-    func updateContent(forceShowTimestamp: Bool, widthConstraint: CGFloat) -> SlideDirection? {
+    func uppdateContent(widthConstraint: CGFloat) -> Bool {
```

Comment:
as it doesn't do the update, maybe 
```suggestion
    func shouldUpdateContent(widthConstraint: CGFloat) -> Bool {
```
describe better what the method does, no?

Reply:
well in this case I think it would be better to keep it as it is and I updated the documentation on top of the method to better describe what that method does.

Reply:
I disagree, the method name is the primary piece of documentation, comments are secondary: everyone will read the method name but not everyone will read the doc comment, so better to be precise with the name.

Reply:
@agisilaos I think we should let the person who started the conversation resolve the conversation, otherwise the discussion can be closed prematurely.

Reply:
> as it doesn't do the update, maybe
> 
> describe better what the method does, no?

@netbe Done!